### DISCUSSION TITLE
Apps not working after deploying

### DISCUSSION THREAD

**Discussion by aaa11ever** [vercel/community#1852] [Help] [answered]
### Summary

I tried to deploy python apps and node.s apps but nothing is working for me. 
One way or other all apps show errors,  can anyone help me in installation of apps. 

      I will be grateful for your help.

**Comment by amyegan** [comment]
Hi @aaa11ever! Can you share a public repo or [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example)? We need more information to help you identify the cause of this problem 🙏

**Comment by aaa11ever** [comment]
Hello,

These are the apps, i try to run but noneof them works.

https://github.com/mohamadkhalaj/Demo-TradingPlatform.git
https://github.com/johnegus/robinhoodClone.git
https://github.com/suenot/profitmaker.git

So please investigate the issue.

   Hope to hear from you soon.

**Comment by amyegan** [comment, marked as answer]
I took a look at the projects you shared, and they seem to be designed for a traditional server environment rather than a serverless environment. That seems to be why they are not working. We have a few resources that can help you set up projects for serverless:
- [Serverless Functions Overview](https://vercel.com/docs/concepts/functions/serverless-functions#)
- [Using Express.js with Vercel](https://vercel.com/guides/using-express-with-vercel)
- [Using the Python Runtime with Serverless Functions](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/python#)
